### DISCUSSION TITLE
Is there an example for building Windows ARM64 on Github Actions

### DISCUSSION THREAD

**Discussion by aminalaee** [pypa/cibuildwheel#1334] [Q&A] [answered]
I've tried to build Windows ARM64 on an Intel machine, but it is failing:
https://github.com/microsoft/picologging/actions/runs/3385338582/jobs/5623366825

It seems to work fine locally but maybe something different is needed on Github actions?
https://github.com/microsoft/picologging/pull/130#issuecomment-1301447960

This is probably not a cibuildwheel issue, but anyway do you have any examples or suggestion to make it work on Github Actions?

**Comment by henryiii** [comment, marked as answer]
> cibuildwheel version 2.6.1

? I'd at least update first. The failure looks odd in the logs, and pretty sure 2.6.1 didn't have ARM or Python 3.11 support. Something is off there.

**Reply by aminalaee** [reply]
Ok thanks, that was silly mistake. Let me update and see if it's resolved.

**Reply by aminalaee** [reply]
Thanks @henryiii it was indeed my stupid mistake, update fixed it.
